Gemfile.lock

```diff
@@ -760,9 +760,3 @@ DEPENDENCIES
   whenever
   wicked_pdf
   wkhtmltopdf-binary
-
-RUBY VERSION
-   ruby 2.1.5p273
-
-BUNDLED WITH
-   1.15.1
```

Comment:
Hi @lauraturk I'm very excited to have you on board!

This is quite similar to what happened to @letakeane in https://github.com/openfoodfoundation/openfoodnetwork/pull/1701#discussion_r127962621.

If you didn't edit the file yourself there's something weird in your setup :thinking: First, I would suggest you do what @enricostano said to @letakeane. Run:

```shell
git reset @^ # Undo last commit leaving unstaged changes
git checkout Gemfile.lock # Remove changes in the Gemfile.lock
git add -p # Stage your changes chunk by chunk ensuring you don't touch the Gemfile.lock
git commit # Well yes, let's commit

# As you modified the last commit you will have to force the push
# (avoid it when collaborating on a branch. More details in
# https://developer.atlassian.com/blog/2015/04/force-with-lease/)
git push --force-with-lease 
```

Then, it would be good to know which versions of bundler and rubygems you have installed. What do the following commands return for you?

```shell
bundler -v
gem --version
```